### PR TITLE
WTB-382 admission service swagger

### PR DESCRIPTION
**Pull request by darrylhebbes**
FIRST PR! (draft)
This is a draft pull request. 

- Descriptions may need to be more accurate
- Can be tested by running `nps start` and going to http://localhost:8000/documentation.html 
- Content Security Policy compromised to allow execution of swagger UI. Needs discussion on allowing this.

### CODE REVIEW

File: src/api/app.ts [hunk 1]
```diff
@@ -11,12 +13,50 @@ import { admissionConfigRouter } from './router/admissionConfigRouter';
...
-  app.use(helmet());
+
+  // DISCUSSME: Extends Helmet's default CSP to allow Swagger-UI to execute REST
+  // calls to localhost, I do not like it. We can test on staging.
+  app.use(
+    helmet({
+      contentSecurityPolicy: {
+        directives: {
+          ...helmet.contentSecurityPolicy.getDefaultDirectives(),
+          'default-src': ["'self'", "'unsafe-inline'", localHostUrl],
```
Comment (pavel-kuznetsov-it) [inline src/api/app.ts:55]: nice trick 👍

**Review by pavel-kuznetsov-it — APPROVED** [review]
Good job! It's much more than the "minimal solution" for sure (e.g 404 and 500 errros descriptions are cool to have but not necessary), but i appreciate the dedication to have it all properly documented 💯 
I think this can be a solution that will work well for this MS.

**Review by cemusta — APPROVED** [review]

File: src/api/app.ts [hunk 2]
```diff
@@ -11,12 +13,63 @@ import { admissionConfigRouter } from './router/admissionConfigRouter';
...
+
+const definition = {
+  openapi: '3.0.3',
+  info: {
+    title: 'Admission Service', // required
+    version: '0.0.1', // required
+    description: 'Admission Service',
+  },
+  servers: [
+    {
```
Comment (cemusta) [inline src/api/app.ts:31]: we can make this coming from an env param later on. so staging doc will only show staging server...

Comment (darrylhebbes) [reply src/api/app.ts:31]: A agree, this had crossed my mind

**Pull request merged**